content/drone-plugins/drone-gcr/index.md

```diff
@@ -8,6 +8,73 @@ logo: google_gcr.svg
 image: plugins/gcr
 ---
 
-{{% alert error %}}
-We are looking for volunteers to port this plugin to drone 0.5
+{{% alert info %}}
+The GCR plugin is an extension around the Docker plugin. Please see the documentation for the Docker plugin for other configuration options.
```

Comment:
Not sure if I should hard link to the Docker plugin docs, or just let people find it.

Reply:
You should link with the help of Hugo links, not a hard link :)